coding/json-common/src/main/java/org/n52/sos/encode/json/AbstractSosResponseEncoder.java

```diff
@@ -49,8 +50,11 @@
  */
 public abstract class AbstractSosResponseEncoder<T extends AbstractServiceResponse> extends JSONEncoder<T> {
     public AbstractSosResponseEncoder(Class<T> type, String operation) {
-        super(type, new OperationEncoderKey(SosConstants.SOS, Sos2Constants.SERVICEVERSION, operation,
-                MediaTypes.APPLICATION_JSON));
+        super(type,
+                new OperationEncoderKey(SosConstants.SOS, Sos2Constants.SERVICEVERSION, operation,
+                        MediaTypes.APPLICATION_JSON),
+                new OperationEncoderKey(SosConstants.SOS, Sos1Constants.SERVICEVERSION, operation,
```

Comment:
The JSON de-/encoding are only implemented for SOS 2.0 and O&M 2.0. The same for the JSON schemas.

So I think the JSON AbstractSosResponseEncoder should not support SOS 1.0.0.